### ISSUE TITLE
how was the ticker file generated?

### ISSUE THREAD

**Issue by FlorinAndrei** [PlatorSolutions/quarterly-earnings-machine-learning-algo#1]
I am condensing all code into a Jupyter notebook that attempts to do everything from scratch in one go, mostly as a coding exercise.

How was `ticker.txt` created, and how can it be re-generated and updated from scratch?

**Comment by Ben-Sherman** [comment]
It was pulled from https://www.sec.gov/include/ticker.txt

Can you add your Jupyter notebook to the repo when you're done?

**Comment by FlorinAndrei** [comment]
Currently the notebook is only doing the steps up to and including generating the clean filings. I'll see what comes next as I'm working through your repo (in my spare time). It's heavily edited from your code, not quite a rewrite but changed quite a bit.

I'll upload it when I feel it's in good shape, which should happen in a few days.

I'm already finding some issues with the repo code when running on Windows - the usual stuff, need to use `os.path.normpath()` to fix separators, file names cannot contain the `:` sign, etc.

**Comment by Ben-Sherman** [comment]
Cool, I'm interested to see the edits.

Yeah, I have not tested with Windows, but I have developed and run the code on Mac and Unix environments.

**Comment by FlorinAndrei** [comment]
I'm pretty sure I've mangled the Windows code. I tried to translate back and forth between the original strings and the Windows-compatible file names (lots of characters are forbidden there) and I believe I was losing data in the process.

Anyway, I don't want to block my Windows laptop with this task, and I was spending too much time on this, so I abandoned Jupyter and switched to your plain Python Linux scripts.

I will send you pull requests if I have any additions worth mentioning.

Thanks!
